readme/installation.md

```diff
@@ -0,0 +1,107 @@
+# Installation
+
+Below are installation instructions for certain distributions as well as
+<a href="#manual-installation">Manual installation</a> instructions.
+
+We welcome contributors to maintain missing distributions.
+
+Please note that distributions may lag with the latest bug fixes.  To
+get the latest bug fixes, use the <a href="#manual-installation">Manual
+installation</a> method.
+
+## Manjaro/Arch
+
+```bash
+yay -S input-remapper-git
+sudo systemctl restart input-remapper
+sudo systemctl enable input-remapper
+```
+
+Please note that the above version may lag from the latest version.  See
+the <a href="#manual-installation">Manual installation</a> method to
+install the latest version.
```

Comment:
No, all `-git` packages from the AUR pull from the latest git main. This is always the latest version with all features

Reply:
Also, the comment about the version "lag" just repeats what is written above